### DISCUSSION TITLE
;-; a doubt

### DISCUSSION THREAD

**Discussion by sebastianjnuwu** [discordjs/discord.js#8171] [Q&A] [answered]
Why in the CHAT_INPUT type the letters cannot be capitalized?

**Comment by monbrey** [comment, marked as answer]
Discord enforces this limitation, and does not allow capital letters in slash command names, or the names of their options.

**Reply by sebastianjnuwu** [reply]
Any specific reason for these limitations? ( That's the answer I'm looking for! )

**Reply by monbrey** [reply]
We aren't Discord so we cant say why they made this choice. You could ask on the discord-api-docs repo or in the Discord Developers server.
